js/index.js

```diff
@@ -13,4 +13,26 @@ for (const item of skills){
     skill.innerText = item;
     skillsList.appendChild(skill);
 }
-    
+let messageSection = document.getElementById('messages');
+let messageList = messageSection.querySelector('ul');
+let newMessage = document.createElement('li');
+newMessage.getElementByName('usersName').innerHTML = "<a href=mailto: + newMessage.getElementByName('usersEmail').email>";
```

Comment:
There is no getElementByName method.  It is getElementsByName and returns an array.  This is reported as an error in the console in the debug tools.

Reply:
Could also use querySelector('form[name="leave_message"]')